components/toast_wrapper/toast_wrapper.tsx

```diff
@@ -2,103 +2,95 @@
 // See LICENSE.txt for license information.
 
 import React from 'react';
-import PropTypes from 'prop-types';
-import {FormattedMessage, injectIntl} from 'react-intl';
+import {FormattedMessage, injectIntl, WrappedComponentProps} from 'react-intl';
```

Comment:
I am considering if using the hook ```useIntl()``` will remove the need to the HOC component and thus make it easier to test and type. Do you mind testing with the hook?

Reply:
`ToastWrapper` is a class component, so I can't use hooks. Of course, I can convert it to FC, but I don't think it is worth the effort just to remove HOC. 

Please let me know if I am missing something.

Reply:
@sinansonmez there are a few ways to use hooks within class components one being to use them as a render prop, but we can probably skip that in this case.

Reply:
Oh okay, I didnt know about it. I will check in details. Thanks @nevyangelova